container/example-config/basic.toml

```diff
@@ -1,27 +1,27 @@
-[[agent]]
+[[agents]]
```

Comment:
Why did you change this to plural?

Reply:
Because @maackle convinced me: https://github.com/holochain/holochain-rust/pull/598/files/f1a93b42af7cd2594aa2bfe8736c9ca55c77d86a#r232011685